src/wx/visualboyadvance-m.metainfo.xml

```diff
@@ -20,15 +17,17 @@
   </p>
  </description>
  <screenshots>
-  <screenshot type="default">https://visualboyadvance-m.org/screenshots/1.png</screenshot>
-  <screenshot>https://visualboyadvance-m.org/screenshots/2.png</screenshot>
-  <screenshot>https://visualboyadvance-m.org/screenshots/3.png</screenshot>
-  <screenshot>https://visualboyadvance-m.org/screenshots/4.png</screenshot>
+     <screenshot type="default"><image type="source">https://i0.wp.com/visualboyadvance.org/wp-content/uploads/2023/01/visual-boy-advance-installed.webp</image></screenshot>
+     <screenshot><image type="source">https://i0.wp.com/visualboyadvance.org/wp-content/uploads/2023/01/visualboyadvance-ubuntu-software-store.webp</image></screenshot>
+     <screenshot><image type="source">https://i0.wp.com/visualboyadvance.org/wp-content/uploads/2023/01/screen-filters-vba.webp</image></screenshot>
+     <screenshot><image type="source">https://i0.wp.com/visualboyadvance.org/wp-content/uploads/2023/01/enable-cheats-on-visual-boy-advance.png</image></screenshot>
+     <screenshot><image type="source">https://i0.wp.com/visualboyadvance.org/wp-content/uploads/2023/01/visualboyadvance-on-android.webp</image></screenshot>
```

Comment:
What's wrong with the screenshots before?

Reply:
The only requirement here is to have the <image> tag.
I changed the links because they were broken when I tested them (?!?!), but now I crosschecked again and everything seems fine. I will revert to the original links.

Reply:
I copy/pasted my patch from the release tarball, hence, the mix up. I reverted to the original screenshot links.